cmd/buildah/rm.go

```diff
@@ -27,7 +28,16 @@ var (
 	}
 )
 
+// writeError writes `lastError` into `w` if not nil and return the next error `err`
+func writeError(w io.Writer, err error, lastError error) error {
```

Comment:
Just wondering if this might be better placed in util.go

Reply:
Right now, it's not used other than here.
I can look at `./cmd/buildah/rmi.go` (f.e.) and see if that can be used there too which makes it more reasonable to what you're suggesting.

does that sound OK?

Reply:
👍 That works for me @ripcurld0  Just keep it in the back of your mind and if you find a use for it somewhere else, moving it to util at that time would be fine.  Thanks!